api/app/Jobs/Middleware/GcNotifyRateLimited.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace App\Jobs\Middleware;
+
+use Closure;
+use Illuminate\Support\Facades\RateLimiter;
+
+class GcNotifyRateLimited
+{
+    /**
+     * Process the queued job.
+     *
+     * @param  \Closure(object): void  $next
+     */
+    public function handle(object $job, Closure $next): void
+    {
+        $executed = RateLimiter::attempt(
+            'gcnotify_api',
+            $perMinute = config('notify.client.max_requests_per_minute'),
```

Comment:
@petertgiles the [Laravel docs](https://laravel.com/docs/10.x/rate-limiting#basic-usage) use this in their example, but it seems very strange to define a variable that the linter flags as unused and when the type is an `int`. Am I a PHP 🦖 and am I missing something here?

## Screenshots

<img width="549" alt="Screen Shot 2024-05-29 at 10 23 36" src="https://github.com/GCTC-NTGC/gc-digital-talent/assets/3046459/29198fa6-433d-401a-8d31-b8de4f10ee9d">
<img width="782" alt="Screen Shot 2024-05-29 at 10 23 29" src="https://github.com/GCTC-NTGC/gc-digital-talent/assets/3046459/6465b524-f9db-4fea-8686-f619a1dd82d3">

Reply:
Yeah, Laravel sometimes does things in the documentation that the linter doesn't like.  I suppose they're still shoring up their type hinting.  Typed PHP is still relatively young I suppose.

Reply:
You're welcome to try out experimenting with the GCNOTIFY_MAX_REQUESTS_PER_MINUTE variable to see how it works.